ClosedOpporunities.cls

```diff
@@ -0,0 +1,23 @@
+public with sharing class ClosedOpporunities{
+    
+    /**Aggregate function to display closed won opportunity records**/
+    public void closedOpportunity()
+    {
+        AggregateResult[] listedClosedOpportunities = [Select sum(Amount) 
+                                                       FROM Opportunity 
+                                                       WHERE StageName in ('Closed Won','Closed Lost') and CloseDate = THIS_FISCAL_YEAR];
+        String currentFiscalYear = [SELECT FiscalYearSettings.Name FROM Period WHERE Type = 'Year' AND StartDate <= TODAY AND EndDate >= TODAY].FiscalYearSettings.Name;
+        System.debug('Current fiscal year '+currentFiscalYear );
+       
+        Map<String,Double> mapFiscalYear = new Map<String,Double>();
```

Comment:
@Manisha7992 Purpose of using Map here is query should be group by Fiscal year and map should contain value of each fiscal year.